tensorboard/webapp/feature_flag/effects/feature_flag_effects.ts

```diff
@@ -77,6 +80,17 @@ export class FeatureFlagEffects {
     {dispatch: false}
   );
 
+  readonly storeFeatureFlag$ = createEffect(
```

Comment:
Mind adding some documentation for the effect? I'd like us to get into the habit of documenting effects like they were functions.

Reply:
That comment is good enough for me. Thanks!